app/assets/stylesheets/application.css

```diff
@@ -29,6 +29,17 @@ body, footer, main, .front-section {
     width: 100%;
 }
 
+.banner {
+    position: absolute;
+    top: 50px;
+    left: 0px;
+    right: 0px;
+    width: auto;
+    height: 100%;
+    z-index: -1;
+    filter: brightness(50%);
```

Comment:
In my opinion you should apply the filter directly to the image data instead of leaving it to the browser, since this is the only place where we need it, so we might as well make sure the text doesn't become completely illegible on non-supported browsers.

Additionally, the current layouting code causes overflow on my device (1265px width) and at the bottom, the orange shows through. Working with height 100% here doesn't make much sense unless you having a surronding relatively positioned element that is exactly the height that you want. Right now, the image will scale when I vertically resize my browser.
The best way to go would probably be a `background-image` with `background-size: cover` and then make sure you kill all the marging around the topmost section so that no orange shows through.